<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="pbFacebook">
      <property name="text">
       <string>Facebook</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pbMessaging">
      <property name="text">
       <string>Messaging</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pbKingdom">
      <property name="text">
       <string>Kingdom</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pbStore">
      <property name="text">
       <string>Store</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QComboBox" name="cbLogin"/>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLineEdit" name="leUsername"/>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLineEdit" name="lePassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="PersonalInfoFrame" name="personalInfoFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pbAddUser">
      <property name="text">
       <string>Add user</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pbRemoveUser">
      <property name="text">
       <string>Remove user</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="0" column="3">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PersonalInfoFrame</class>
   <extends>QFrame</extends>
   <header>PersonalInfoFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leUsername</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>pbAddUser</tabstop>
  <tabstop>pbRemoveUser</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>pbFacebook</tabstop>
  <tabstop>pbMessaging</tabstop>
  <tabstop>pbKingdom</tabstop>
  <tabstop>pbStore</tabstop>
  <tabstop>cbLogin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbAddUser</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotLogin()</slot>
 </slots>
</ui>
